terraform/app/container-definitions/web_container_definition.json

```diff
@@ -209,10 +209,6 @@
       {
         "name": "ALGOLIA_WRITE_API_KEY",
         "value": "${algolia_write_api_key}"
-      },
```

Comment:
We probably do not need this file anymore @saliceti

Reply:
@david-mears-dfe can you remove the whole `terraform/container-definitions` directory please?